storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,34 @@
+[id="persistent-storage-csi-azure-disk"]
+= Azure Disk CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+
+:FeatureName: Azure Disk Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure Disk storage assets, {product-title} installs the Azure Disk CSI Driver Operator and the Azure Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
I'd be careful here, as the CSI Driver and Operator aren't installed by default. The wording implies that it is. The administrator must enable the `TechPreviewNoUpgrade` feature gate to have access to this driver and operator.

Reply:
Agree with @huffmanca , I'd like to add how to install the csi driver and operator here. And add a note about if we used the `TechPreviewNoUpgrade` featureset to install csi driver and operator, it's undone and will block the upgrade.